### PR TITLE
feat(components): only show the navigation buttons on Lightbox when there is more than 1 image

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations
Don't show the navigation buttons when there is only one image in the Lightbox, avoids confusion to the user.

### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by chris-at-jobber — COMMENTED** [review]
Design approved ✅ 

One thought on the docs themselves, could we get rid of the 😈 emoji from the buttons and use more realistic example images? Maybe some [landscaping imagery from Unsplash?](https://unsplash.com/s/photos/landscaping)

**Comment by rodrigoeidelvein** [comment]
> Design approved ✅
> 
> One thought on the docs themselves, could we get rid of the 😈 emoji from the buttons and use more realistic example images? Maybe some [landscaping imagery from Unsplash?](https://unsplash.com/s/photos/landscaping)

For sure, I'm starting to have nightmares with the images

**Comment by chris-at-jobber** [comment]
Love it! Will leave code review to the experts 😄

File: packages/components/src/LightBox/LightBox.tsx [hunk 1]
```diff
@@ -152,8 +152,13 @@ export function LightBox({
...
           </div>
-          <PreviousButton onClick={debouncedHandlePrevious} />
-          <NextButton onClick={debouncedHandleNext} />
+
+          {images.length > 1 ? (
+            <>
+              <PreviousButton onClick={debouncedHandlePrevious} />
+              <NextButton onClick={debouncedHandleNext} />
+            </>
+          ) : null}
```
Comment (darryltec) [inline packages/components/src/LightBox/LightBox.tsx:161]:
```suggestion
          {images.length > 1 && (
            <>
              <PreviousButton onClick={debouncedHandlePrevious} />
              <NextButton onClick={debouncedHandleNext} />
            </>
          )}
```

should suffice, no?

File: packages/components/src/LightBox/LightBox.stories.mdx [hunk 2]
```diff
@@ -19,17 +19,18 @@ import { LightBox } from "@jobber/components/LightBox";
...
+    url: "https://images.unsplash.com/photo-1597201278257-3687be27d954?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80",
+    caption:
+      "This was the view of Bushart Gardens in Victoria, BC, Canada in July from a hill.",
   },
   {
-    title: "😮",
-    url: "https://i.imgur.com/OsLxYkT.jpg",
+    title: "A house",
+    url: "https://images.unsplash.com/photo-1592595896616-c37162298647?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80",
     caption: "Suprised face",
```
Comment (darryltec) [inline packages/components/src/LightBox/LightBox.stories.mdx:30]: Is this an easter egg? 😆

**Review by darryltec — APPROVED** [review]
LGTM

**Pull request merged**